projects/igniteui-angular/migrations/update-18_1_0/changes/members.json

```diff
@@ -24,6 +24,16 @@
             "definedIn": [
                 "IgxPaginatorComponent"
             ]
```

Comment:
Since 18.1 is already released these migrations should be moved to a 18.2 migration